### PR TITLE
Initalize components

### PR DESCRIPTION
**Pull request by bilalrajput09**
## Bookstore: initialize project with components:

- Contains individual book state, as well as a button to delete the book.
- Contains a list to renders individual books.
- Contains a form to create new books.
- Contains a navigation to link to the 2 routes you created.

### CODE REVIEW

File: src/Components/BookList.js
Comment (KDlamini) [inline src/Components/BookList.js:1]: Great job setting up components with React!!! 👏🏾 💯

According to the project requirements, you must create an individual book state that represents a single book. Preferably, you should name this component `Book.js` but you can give it any reasonable name. In your case, the `BookList` component should render the `Book` component.

image

1. This means the `<Book>` takes the book data as a prop and renders the title and the author.

For example:

```jsx
const Book = ({ book })  => {

  return (
    <article>
        <h2>{book.title}</h2>
        <h3>{book.author}</h3>
        <p>{book.category}</p>
        <ul>
          <li>
            <button type="button">Comments</button>
          </li>
          <li>
            <button
              type="button"
              onClick={() => {}}
            >
              Remove
            </button>
          </li>
          <li>
            <button type="button">Edit</button>
          </li>
        </ul>
    </article>
  );
}

export default Book;
```

2. Then the `<BookList>` component renders a list of books using the `<Book>` component. Notice how I used `semantic tags`, `<article>` to represent one book section, and `<section>` to represent a group of books.

For example:

```jsx
import Book from './Book'

const BookList = (props) => {
  const { books } = props;

  return (
    <section>
      {books.map((book) => (
        <Book
          key={book.item_id}
          book={book}
        />
      ))}
    </section>
  );
};

export default BookList;
```

I hope this makes sense 😊

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @bilalrajput09 🙋‍♀️,  

Great work so far! 💯 
You have done a great job initializing the Bookstore with components. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR title and description
✔️  Routes are correctly set up using [React Router](https://github.com/remix-run/react-router/tree/dev/examples/basic)  in `<App>`.
✔️  The `<Navbar>` contains 2 `<Link>`s. One to `/` and one to `/categories`
✔️  The 2 page components: `<Books>` and `<Categories>` are rendered correctly

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/BookList.js
Comment (bilalrajput09) [reply src/Components/BookList.js:1]: Resolved ✅✅

**Review by AnselemOdims — APPROVED** [review]
### Hi  @bilalrajput09 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**